dev/source/docs/learning-ardupilot-uarts-and-the-console.rst

```diff
@@ -5,35 +5,126 @@ UARTs and the Console
 =====================
 
 A lot of components in ArduPilot rely on UARTs. They are used for debug
-output, telemetry, GPS modules and more. Understanding how to talk to
+output, telemetry, GPS modules, and more. Understanding how to talk to
 the UARTs via the HAL will help you understand a lot of ArduPilot code.
 
 The 8 UARTs
 ===========
 
-The ArduPilot HAL currently defines 8 UARTs. The HAL itself doesn't
-define any particular roles for these UARTs, but the other parts of
-ArduPilot assume they will be assigned particular functions
-
--  uartA - the console (usually USB, runs MAVLink telemetry)
--  uartB - the first GPS
--  uartC - primary telemetry (telem1 on most autopilots)
--  uartD - secondary telemetry (telem2 on most autopilots)
--  uartE - 2nd GPS
--  uartF - User Configurable
--  uartG - User Configurable
--  uartH - User Configurable
-
-See also :ref:`sitl-serial-mapping`
+The ArduPilot HAL currently defines 8 UARTs. The HAL itself does not define any particular roles for these UARTs, but the other parts of ArduPilot assume they will be assigned particular functions. The command-line options for using with sim_vehicle.py the serial port should be preceded by :code:`-A` to pass along to the vehicle binary. Make sure to include the :code:`uart` protocol. Specifying a baudrate is not required, but is more consistent. For example, :code:`sim_vehicle.py --console --map -A --uartF=uart:/dev/ttyS15:115200`.
+
```

Comment:
Maybe add a note that we can now using --serialX instead of --uartX as this is more convenient

Reply:
Fixed!